lib/Ama/Model/Votes.pm

```diff
@@ -42,10 +42,10 @@ sub _remove {
   my $sql;
   my $results = eval { 
       if ($entry_type eq 'questions'){ #if it's a question
-        $sql = 'delete from questions where question_id = ?' #delete it from questions table
+        $sql = 'delete from questions where question_id = ?' #hide it from questions table
```

Comment:
The _remove function does indeed delete the question.  It's very misleading and actually wrong to specify a comment that `delete from questions ...` will hide the question.  The comment refers to this code, this code will delete the question.  Elsewhere, where you used to call _remove is commented out, so _remove is not called.  But _remove still removes questions.